src/providers/mdal/qgsmdalprovider.cpp

```diff
@@ -604,4 +604,22 @@ QGISEXTERN void fileMeshFilters( QString &fileMeshFiltersString, QString &fileMe
   QgsMdalProvider::fileMeshFilters( fileMeshFiltersString, fileMeshDatasetFiltersString );
 }
 
+/**
+  Returns whether MDAL can load this file as mesh
+
+  \since QGIS 3.6
+*/
+QGISEXTERN bool isValidMeshFileName( const QString &fileName )
+{
+  bool valid( false );
+  QByteArray curi = fileName.toAscii();
```

Comment:
This will probably not work well with non-ascii filenames (e.g. `čůčo.2dm`) - can you check that it works? (also in mdal provider)

Reply:
ok, the QGIS project is not mature enough for "čůčo.2dm", but I added "štvôrček.2dm"